assets/src/settings/index.js

```diff
@@ -56,50 +66,51 @@ class Settings extends Component {
 		} = this.state;
 
 		return (
-			<div className="ncc-page">
-				<h1>{ __( 'Content Conversion Settings' ) }</h1>
-				<br />
-
-				<p>
-					{ __(
-						'Adding content to the queue, enables it to be converted to Gutenberg Blocks. The queue is also a backup point for possible reverting.'
-					) }
-				</p>
-				<p>
-					{ __(
-						'Existing HTML content is first selected by type, and added to a conversion queue. Queued content is then converted.'
-					) }
-				</p>
-				<h3>{ __( 'Specify content type' ) }</h3>
-				<ul>
-					<li>
-						content types:{' '}
-						<input type="text" disabled={ true } value={ conversionContentTypesCsv } />
-					</li>
-					<li>
-						content statuses:{' '}
-						<input type="text" disabled={ true } value={ conversionContentStatusesCsv } />
-					</li>
-				</ul>
-
-				<h3>{ __( 'Conversion params' ) }</h3>
-				<ul>
-					<li>
-						conversion batch size:{' '}
-						<input type="text" disabled={ true } value={ conversionBatchSize } />
-					</li>
-				</ul>
-
-				<h3>{ __( 'Queued stats' ) }</h3>
-				<ul>
-					<li>
-						<b>
-							<u>{ queuedEntries }</u>
-						</b>{' '}
-						posts are currently selected and queued for conversion
-					</li>
-				</ul>
-			</div>
+			<Fragment>
+				<div className="newspack-logo-wrapper">
+					<NewspackLogo />
+				</div>
+				<Grid>
+					<FormattedHeader
+						headerIcon={ <SettingsIcon /> }
+						headerText={ __( 'Content conversion settings' ) }
+						subHeaderText={ __( 'Adding content to the queue to convert it to Gutenberg Blocks.' ) }
```

Comment:
Proposing change in text:
Specify type of content to be converted to Gutenberg Blocks.

Reply:
I think, being quite broad here works better since you will be able to choose what type of content you want to convert (post, pages, CPT).

A subheader should be short and concise and not too descriptive.